histomicstk/cli/SuperpixelSegmentation/CreateDataset.py

```diff
@@ -0,0 +1,615 @@
+import os
```

Comment:
@slee172 

The name of the CLI's python/xml file and its parent directory should match.